documenting persistence with Postgres in SDK docs (#722)
digital-asset/daml: docs/source/tools/sandbox.rst

Here, ``da run sandbox --`` tells the SDK Assistant to run ``sandbox`` from the active SDK release and pass it any arguments that follow. The example passes the DAR file to load (``Main.dar``) and the optional ``--scenario`` flag tells Sandbox to run the ``Main:example`` scenario on startup. The scenario must be fully qualified; here ``Main`` is the module and ``example`` is the name of the scenario, separated by a ``:``. The scenario is used for testing and development; it is not run in production.

Running with persistence
************************

> By default the Sandbox uses an in-memory store and as such it loses its state when stopped or restarted. There is a possibility to use a Postgres database for persistence, which allows the Sandbox to be shut down and continued later where it left off.

Reviewer: ```suggestion
By default, Sandbox uses an in-memory store, which means it loses its state when stopped or restarted. If you want to keep the state, you can use a Postgres database for persistence. This allows you to shut down Sandbox and start it up later, continuing where it left off.
```

> The application has to have access to an initially empty database, and the database user needs to have authority to execute DDL operations as the Sandbox manages its own database schema, applying migrations if necessary when upgrading versions. 

Reviewer: ```suggestion
To set this up, you must:
- create an initially empty Postgres database that the Sandbox application can access 
- have a database user for Sandbox that has authority to execute DDL operations 

  This is because the Sandbox manages its own database schema, applying migrations if necessary when upgrading versions. 
```

> In order to start the Sandox using persistence you need to pass an ``--jdbcurl <value>`` option, where `<value>` is a valid jdbc url containing the username, password and database name to connect to. Here is an example for such a url: ``jdbc:postgresql://localhost/test?user=fred&password=secret``

Reviewer: ```suggestion
To start Sandbox using persistence, pass an ``--jdbcurl <value>`` option, where `<value>` is a valid jdbc url containing the username, password and database name to connect to.

Here is an example for such a url: ``jdbc:postgresql://localhost/test?user=fred&password=secret``
```

> More on jdbc urls can be found here: https://jdbc.postgresql.org/documentation/head/connect.html

Reviewer: ```suggestion
If you're not familiar with JDBC URLs, see the JDBC docs for more information: https://jdbc.postgresql.org/documentation/head/connect.html
```

Command-line reference
**********************

> You can start the Sandbox by running: ``sandbox [options] <archive>...`` All the available options can be seen by running ``da run sandbox -- --help``

Reviewer: Making things more active and instructing:

```suggestion
To start Sandbox, run: ``sandbox [options] <archive>...``

To see all the available options, run ``da run sandbox -- --help``
```

Here, ``da run sandbox --`` tells the SDK Assistant to run ``sandbox`` from the active SDK release and pass it any arguments that follow. The example passes the DAR file to load (``Main.dar``) and the optional ``--scenario`` flag tells Sandbox to run the ``Main:example`` scenario on startup. The scenario must be fully qualified; here ``Main`` is the module and ``example`` is the name of the scenario, separated by a ``:``. The scenario is used for testing and development; it is not run in production.

Running with persistence
************************

By default, Sandbox uses an in-memory store, which means it loses its state when stopped or restarted. If you want to keep the state, you can use a Postgres database for persistence. This allows you to shut down Sandbox and start it up later, continuing where it left off.

To set this up, you must:
- create an initially empty Postgres database that the Sandbox application can access 
- have a database user for Sandbox that has authority to execute DDL operations 

>   This is because the Sandbox manages its own database schema, applying migrations if necessary when upgrading versions. 

Reviewer: ```suggestion
  This is because Sandbox manages its own database schema, applying migrations if necessary when upgrading versions. 
```

> To start Sandbox using persistence, pass an ``--jdbcurl <value>`` option, where `<value>` is a valid jdbc url containing the username, password and database name to connect to.

Reviewer: ```suggestion
To start Sandbox using persistence, pass an ``--jdbcurl <value>`` option, where ``<value>`` is a valid jdbc url containing the username, password and database name to connect to.
```
